Frameworks/CoreText/DWriteWrapper_CoreText.mm

```diff
@@ -52,10 +46,26 @@ static DWRITE_TEXT_ALIGNMENT __CoreTextAlignmentToDwrite(CTTextAlignment alignme
     }
 }
 
+static DWRITE_WORD_WRAPPING __CTLineBreakModeToDWrite(CTLineBreakMode lineBreakMode) {
```

Comment:
_How do the unit tests build?_
not sure what this question means, or what answer you're fishing for. probably something like "CoreGraphics.UnitTests links agains CoreGraphics dll"?

_Are these items in a .def file?_
yes

_Would a private dll export have any other potential pitfalls with regards to encapsulation and versioning?_
i mean, we deliberately broke encapsulation here. versioning risks... _ is intended to communicate the same level of privacy as the stuff in CFFoundationInternal.h, so I figured we'd just continue whatever risk we accepted already, but yeah because of this we'd have to make sure unit tests, coregraphics, coretext, and foundation, all get a version update if the contract of these functions changes.

---
In reply to: [90937900](https://github.com/Microsoft/WinObjC/pull/1504#discussion_r90937900) [](ancestors = 90937900,90936774,90918451,90746929,90742638)